lesson_2/Lesson_2.4.ipynb

```diff
@@ -0,0 +1,310 @@
+{
```

Comment:
<p>Can also just compute the mean -- less confusing</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/AlexAndorra/notebooks-experiments/pull/2/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>

Reply:
<p>I thought this would be more explicit so if folks are newer they can see copy and paste the numerator and denominator themselves to see what happened, rather than implicitly in a function</p><p> </p>

Reply:
<p>I see, that's fine by me too</p>